luda-editor/luda-editor-client/src/app/sequence_list/types/sequence_titles_load_state.rs

```diff
@@ -0,0 +1,14 @@
+use std::time::Duration;
+
+#[derive(Clone)]
+pub enum SequenceTitlesLoadStateDetail {
+    Loading,
+    Loaded { titles: Vec<String> },
+    Failed { error: String },
+}
+
+#[derive(Clone)]
+pub struct SequenceTitlesLoadState {
+    pub started_at: Duration,
```

Comment:
I think it's kind of over-engineering to handle `started_at` for loading sequence titles, because it only called once, not multiple time at once. How do you think about it?

Reply:
I think user can click `Reload titles` button multiple time at once.